github/changed-files/action.yml

```diff
@@ -0,0 +1,35 @@
+---
+name: changed-files
+description: |
+  Check the files changed between two git references.
+  It uses the `git diff` command to get the files changed between two git references.
+  It can filter the files based on the file extension.
+  The code is based on the [git-changed-files](https://github.com/kandhavivekraj/git-changed-files) with minimal changes.
+inputs:
+  baseRef:
```

Comment:
we use `base-ref` instead of camelcase

Reply:
I don't know why we adopted that variable-definition pattern instead of the snake or camel cases, which are used in all programming languages. What is exactly the reason we chose it, and where is the best practice or documentation about it? I will change it, but I want to know why we do it this way.

Reply:
because GH actions use that approach

Reply:
See the documentation in https://docs.github.com/en/actions/sharing-automations/creating-actions/creating-a-composite-action

```yml
inputs:
  who-to-greet:  # id of input
    description: 'Who to greet'
```

or https://docs.github.com/en/actions/sharing-automations/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow

```yml
    inputs:
      config-path:
        required: true
```

and because we discussed this back in the day in an internal HG issue to actually use that naming convention here

Reply:
We probably need to update:
- https://github.com/elastic/oblt-actions/blob/main/docs/CONTRIBUTING.md